Tools/baselineDef.cc

```diff
@@ -154,44 +160,67 @@ void BaselineVessel::prepareDeepTopTagger()
-  AK4Inputs.addSupplamentalVector("qgMult",                              qgMult_f);
-  //AK4Inputs.addSupplamentalVector("qgMult",                             tr->getVec<float>(UseNoLepVar("qgMult")));
-  AK4Inputs.addSupplamentalVector("recoJetschargedHadronEnergyFraction", tr->getVec<float>(UseNoLepVar("recoJetschargedHadronEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetschargedEmEnergyFraction",     tr->getVec<float>(UseNoLepVar("recoJetschargedEmEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetsneutralEmEnergyFraction",     tr->getVec<float>(UseNoLepVar("recoJetsneutralEmEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetsmuonEnergyFraction",          tr->getVec<float>(UseNoLepVar("recoJetsmuonEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetsHFHadronEnergyFraction",      tr->getVec<float>(UseNoLepVar("recoJetsHFHadronEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetsHFEMEnergyFraction",          tr->getVec<float>(UseNoLepVar("recoJetsHFEMEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("recoJetsneutralEnergyFraction",       tr->getVec<float>(UseNoLepVar("recoJetsneutralEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("PhotonEnergyFraction",                tr->getVec<float>(UseNoLepVar("PhotonEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("ElectronEnergyFraction",              tr->getVec<float>(UseNoLepVar("ElectronEnergyFraction")));
-  AK4Inputs.addSupplamentalVector("ChargedHadronMultiplicity",           tr->getVec<float>(UseNoLepVar("ChargedHadronMultiplicity")));
-  AK4Inputs.addSupplamentalVector("NeutralHadronMultiplicity",           tr->getVec<float>(UseNoLepVar("NeutralHadronMultiplicity")));
-  AK4Inputs.addSupplamentalVector("PhotonMultiplicity",                  tr->getVec<float>(UseNoLepVar("PhotonMultiplicity")));
-  AK4Inputs.addSupplamentalVector("ElectronMultiplicity",                tr->getVec<float>(UseNoLepVar("ElectronMultiplicity")));
-  AK4Inputs.addSupplamentalVector("MuonMultiplicity",                    tr->getVec<float>(UseNoLepVar("MuonMultiplicity")));
-  AK4Inputs.addSupplamentalVector("DeepCSVb",                            tr->getVec<float>(UseNoLepVar("DeepCSVb")));
-  AK4Inputs.addSupplamentalVector("DeepCSVc",                            tr->getVec<float>(UseNoLepVar("DeepCSVc")));
-  AK4Inputs.addSupplamentalVector("DeepCSVl",                            tr->getVec<float>(UseNoLepVar("DeepCSVl")));
-  AK4Inputs.addSupplamentalVector("DeepCSVbb",                           tr->getVec<float>(UseNoLepVar("DeepCSVbb")));
-  AK4Inputs.addSupplamentalVector("DeepCSVcc",                           tr->getVec<float>(UseNoLepVar("DeepCSVcc")));
+  AK4Inputs.addSupplamentalVector("qgMult", qgMult_f);
+  
+  // loop over variables and add to AK4Inputs
+  for (const auto& variable : AK4Variables)
+  {
+    AK4Inputs.addSupplamentalVector(variable, tr->getVec<float>(UseCleanedJetsVar(variable)));
+  }
 
+  // ----- AK8 Jets -----
```

Comment:
remove commented block

Reply:
Yep. I was thinking of using it at first, but there is no "add supplemental vector" for AK8 jets. It is gone now.